### PR TITLE
Wrote in Race doc

### PR DESCRIPTION
**Pull request by Saberfiend**

### CODE REVIEW

File: docs/lore/races.md [hunk 1]
```diff
@@ -2,4 +2,8 @@
 title: Races - Raven Rock RP
 description: The guide describes how you can contribute to the Raven Rock RP documentation site docs.ravenrockrp.com.
 author: Foxtrek_64
----
\ No newline at end of file
+---
+
+# Races
```
Comment (TheBrambleShark) [inline docs/lore/races.md:7]: This should be "Rases"

**Review by TheBrambleShark — APPROVED** [review]
LGTM

File: docs/lore/races.md (hunk 1 above)
Comment (TheBrambleShark) [reply docs/lore/races.md:7]: Nevermind

Comment (Saberfiend) [reply docs/lore/races.md:7]: Nuh uh

**Pull request closed without merging**